dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/relationship/RelationshipOperationParamsMapper.java

```diff
@@ -62,11 +57,9 @@ class RelationshipOperationParamsMapper {
   public RelationshipQueryParams map(RelationshipOperationParams params)
       throws NotFoundException, ForbiddenException, BadRequestException {
 
-    UserDetails currentUser = CurrentUserUtil.getCurrentUserDetails();
-
     IdentifiableObject entity =
         switch (params.getType()) {
-          case TRACKED_ENTITY -> validateTrackedEntity(currentUser, params.getIdentifier());
+          case TRACKED_ENTITY -> trackedEntityService.getTrackedEntity(params.getIdentifier());
```

Comment:
Is this call as "cheap"/"expensive" as the one we did before `trackedEntityService.getTrackedEntity(uid, null, TrackedEntityParams.FALSE, false)`?

Reply:
It is the same, following the flow we are just getting the exact some tracked entity.